src/benchmarks/micro/libraries/System.Formats.Tar/Perf.Tar.cs

```diff
@@ -0,0 +1,115 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Formats.Tar.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.NoWASM)]
+    public class Perf_TarWriter
+    {
+        private static readonly string _fileName = "file.txt";
+        private static Dictionary<string, string> _ea;
+        private static MemoryStream _memoryStream;
```

Comment:
nit: no need for using static here

```suggestion
        private Dictionary<string, string> _ea;
        private MemoryStream _memoryStream;
```